packages/Toast/src/Toast.js

```diff
@@ -48,23 +48,34 @@ const Toast = React.forwardRef((props, ref) => {
     },
   }));
 
-  const [isToastOpen, setIsToastOpen] = React.useState(isOpen === undefined ? true : isOpen);
+  const isControlled = isOpen !== undefined;
```

Comment:
typeof isOpen === "undefined" is a better approach [readmore](https://thisthat.dev/variable-undefined-vs-typeof-variable-undefined/)

Reply:
interesting!